scripts/exregional_nonvarcldanl.sh

```diff
@@ -312,6 +320,7 @@ cat << EOF > gsiparm.anl
    i_T_Q_adjust=${i_T_Q_adjust},
    l_rtma3d=${l_rtma3d},
    i_precip_vertical_check=${i_precip_vertical_check},
+   i_uncertainty=${i_uncertainty},
```

Comment:
If it only takes the value of "True" or "False", suggest using "l_uncerntainty"

Reply:
Thanks, I switched the type of the variable but forgot to change the name to match.  I'll make that change.